src/patternfly/layouts/Toolbar/toolbar.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="pf-l-toolbar{{#if pf-l-toolbar--modifier}} {{pf-l-toolbar--modifier}}{{/if}}"
+<div class="pf-l-toolbar{{#if toolbar--modifier}} {{toolbar--modifier}}{{/if}}"
```

Comment:
add a `toolbar--attribute` block, and I might also follow up and remove `pf-l-toolbar__id` and just include `id` within `toolbar--attribute`

Reply:
attribute block added, removed pf-l-toolbar__id, it also wasn't being used